boards/x86/x86_jailhouse/Kconfig.defconfig

```diff
@@ -8,6 +8,10 @@ config JAILHOUSE
 	bool "Zephyr port to boot as a (x86) Jailhouse inmate cell payload"
 	default y
 
+config REALMODE
```

Comment:
you need to define REALMODE outside of the board to make it available for others, in arch/x86/Kconfig for example, then just set it in the _defconfig of the jailhouse board.

Reply:
I see, thank you very much for pointing this out.

Reply:
It seems arch/x86/Kconfig is for hardware feature configurations. Can I put this item in arch/x86/core/Kconfig?

Reply:
fine with me

Reply:
Sorry for posting comments before carefully look at the code. 

It seems making more sense to put it in the "Boot Options" in misc/Kconfig, and make it depend on X86. Initially I ignore this file and just thought real mode is for X86 only and should be under arch/x86.